src/selectors/edges.js

```diff
@@ -96,3 +98,20 @@ export const getVisibleEdges = createSelector(
         target: edgeTargets[id] || transitiveEdges.targets[id],
       }))
 );
+
+/**
+ * Returns an node ID where source edge is a parameter
+ */
+export const getEdgeswithParams = createSelector(
+  [getEdgeIDs, getNodeType, getEdgeSources, getEdgeTargets],
+  (edgeIDs, nodeType, edgeSources, edgeTargets) => {
+    const nodes_list = {};
+    for (const edgeID of edgeIDs) {
+      const source = edgeSources[edgeID];
+      const target = edgeTargets[edgeID];
+      if (nodeType[source] === 'parameters' && nodeType[target] === 'task')
+        {nodes_list[target] = true;}
```

Comment:
I think this block needs formatting, it's strange that Prettier didn't format this?

Reply:
there's some problem. This was an old function in the previous parameters ticket which I had ommitted. I don't know how it got passed on. I will remove this entirely again